pkg/cluster/config.go

```diff
@@ -296,3 +297,18 @@ func ValidateCIDR(cidr string) error {
 	}
 	return nil
 }
+
+// GetClusterDNS returns the IP address of the DNS server, given the service CIDR of the cluster
+func GetClusterDNS(iprange string) (string, error) {
```

Comment:
`ipRange` as per Go standards.

Reply:
instead of using ipRange we can use cidr or subnet.  we derive ip address from the subnet/cidr.

Reply:
This comment makes sense to me. @jrvaldes did you take this into account?

Reply:
Yes, I did considered the proposed names and sticked with `ip range`.

Reply:
@jrvaldes In general, networking side we do use CIDR or subnet . In fact what you are getting a subnet ( collection of IPs) and you are trying to extract one IP from it. Its better to name it in conventional way so people can understand while reading the param.

Reply:
Addressed in https://github.com/openshift/windows-machine-config-operator/blob/d789e1fa0768fdcd00a8758a7da54d301d0a5b5b/pkg/cluster/config.go#L305